src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishExtension.kt

```diff
@@ -65,5 +74,17 @@ open class NexusPublishExtension(project: Project) {
         )
     )
 
+    val publicationType: Property<PublicationType> = project.objects.property<PublicationType>().convention(PublicationType.MAVEN)
+
+    val ivyPatternLayout: Property<Action<IvyPatternRepositoryLayout>> = project.objects.property<Action<IvyPatternRepositoryLayout>>()
+    fun ivyPatternLayout(action: Action<IvyPatternRepositoryLayout>) {
+        ivyPatternLayout.set(action)
+    }
```

Comment:
I think this should be configurable on `NexusRepository` instead.

Reply:
Yes that's probably a better place